mta/mta_services_test.go

```diff
@@ -1544,6 +1573,83 @@ var _ = Describe("MtaServices", func() {
 			Ω(err).Should(HaveOccurred())
 		})
 	})
+
+	var _ = Describe("UpdateGlobalParameters", func() {
+
+		parameters := map[string]interface{}{"param1": "value1", "param2": "value2"}
+
+		It("Add new global parameters", func() {
+			mtaPath := getTestPath("result", "temp.mta.yaml")
+
+			jsonRootData, err := json.Marshal(getMtaInput())
+			Ω(err).Should(Succeed())
+			Ω(CreateMta(mtaPath, string(jsonRootData), os.MkdirAll)).Should(Succeed())
+
+			jsonGlobalParametersData, err := json.Marshal(&parameters)
+			Ω(err).Should(Succeed())
+			messages, err := UpdateGlobalParameters(mtaPath, string(jsonGlobalParametersData))
+			Ω(err).Should(Succeed())
+			Ω(messages).Should(BeEmpty())
+
+			oMtaInput := getMtaInput()
+			oMtaInput.Parameters = parameters
+			Ω(mtaPath).Should(BeAnExistingFile())
+			yamlData, err := ioutil.ReadFile(mtaPath)
+			Ω(err).Should(Succeed())
+			oMtaOutput, err := Unmarshal(yamlData)
+			Ω(err).Should(Succeed())
+			Ω(reflect.DeepEqual(oMtaInput, *oMtaOutput)).Should(BeTrue())
+		})
+
+		It("Update existing global parameters", func() {
+			updatedParameters := map[string]interface{}{"param1": "value1", "param2": "value3"}
+
+			mtaPath := getTestPath("result", "temp.mta.yaml")
+
+			jsonRootData, err := json.Marshal(getMtaInput())
+			Ω(err).Should(Succeed())
+			Ω(CreateMta(mtaPath, string(jsonRootData), os.MkdirAll)).Should(Succeed())
+
+			jsonGlobalParametersData, err := json.Marshal(&parameters)
+			Ω(err).Should(Succeed())
+			messages, err := UpdateGlobalParameters(mtaPath, string(jsonGlobalParametersData))
+			Ω(err).Should(Succeed())
+			Ω(messages).Should(BeEmpty())
```

Comment:
Instead of updating you should create the mta with the parameters from the start (in line 1609 before you serialize the mta data, add the parameters to the mta input)